akka-bench-jmh/src/main/scala/akka/actor/ForkJoinActorBenchmark.scala

```diff
@@ -102,6 +152,15 @@ object ForkJoinActorBenchmark {
   final val message = "message"
   final val timeout = 15.seconds
   final val messages = 400000
+
+  // update according to cpu
+  final val cores = 8
```

Comment:
Can't you do it in the setUp method? I think there is some annotation like that in JMH.